FF145Relnote: ToggleEvent.source (#41653)
mdn/content: files/en-us/mozilla/firefox/releases/145/index.md

### APIs

- The {{domxref("ToggleEvent/source", "source")}} property of the {{domxref("ToggleEvent")}} interface is now supported.

>   If a [popover](/en-US/docs/Web/API/Popover_API) was triggered open or closed by an HTML element, such as a {{htmlelement("button")}}, the corresponding event will include the element in this property.

Reviewer: i've flipped the part after the comma ("the corresponding event...") to make it more direct
```suggestion
  If a [popover](/en-US/docs/Web/API/Popover_API) is triggered to open or close by an HTML element such as a {{htmlelement("button")}}, the event's `source` property will contain the element that triggered the popover.
```
